debugger.md

```diff
@@ -0,0 +1,83 @@
+#Preview: Instructions for setting up the .NET Core debugger
+*Updated on  March 1st 2016 - setup works for Linux & Mac.*
+
+***Currently working within Microsoft Corp-Net only.***
+
+Thank you for taking the time to try out our bits. 
+This page gives you detailed instructions on how to get your system running. 
+It will be updated on a regulary basis to always reflect the latest behaviour. Now let's get started.​
+
+####Your Feedback​
+
+Please place your feedback in this onenote [here](https://microsoft.sharepoint.com/teams/DD_VSPlat/Diagnostics/_layouts/15/WopiFrame.aspx?sourcedoc={04f36c35-0332-4e99-85f7-a33a4c2987e1}&action=edit&wd=target%28%2F%2FDogfood%20Feedback.one%7C4713cc8e-053b-4a0c-9828-745429ee7f45%2F2016Feb%20CoreCLR%20Debugging%20on%20Mac%2C%20Linux%2C%20Windows%7C890db52c-40ef-4181-ae03-d66d9ee0fe54%2F%29). Simply add a new line for every finding. 
```

Comment:
I would suggest we create a DL since not everyone will likely have access to the OneNote.